Flutter/pdf-viewer/right-to-left.md

```diff
@@ -120,4 +120,6 @@ Then, import the `flutter_localizations` library, specify [localizationsDelegate
   }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
```

Comment:
<img width="162" alt="image" src="https://github.com/user-attachments/assets/a6113ae4-e92a-4097-ade3-55c8a6ac1aad">

Please use API linking or use lower case if using in words

Reply:
Added API links